pkg/server/status.go

```diff
@@ -2520,6 +2523,7 @@ func (s *systemStatusServer) rangesHelper(
 		return serverpb.RangeInfo{
 			Span:          span,
 			RaftState:     convertRaftStatus(raftStatus),
+			RacState:      rac2State,
```

Comment:
nit: add a proto directive to make this named as `RACState`

Reply:
Done
<!-- Reviewable comment -OEU9HAZ2Xhl3SzFYMgf:bb6un1j -->
<!-- Sent from Reviewable.io -->